src/integer_mod_q/poly_over_zq/from.rs

```diff
@@ -78,6 +83,41 @@ impl From<(&PolyOverZ, &Modulus)> for PolyOverZq {
     }
 }
 
+impl From<&ModulusPolynomialRingZq> for PolyOverZq {
+    /// Create a [`PolyOverZ`] from a [`ModulusPolynomialRingZq`].
+    ///
+    /// Parameters:
+    /// - `modulus`: the context polynomial from which the coefficients are copied
+    ///
+    /// # Examples
+    ///
+    /// Returns the context [`PolyOverZq`] representing the modulus object
+    ///
+    /// ```
+    /// use qfall_math::integer_mod_q::{ModulusPolynomialRingZq, PolyOverZq};
+    /// use std::str::FromStr;
+    ///
+    /// let modulus = ModulusPolynomialRingZq::from_str("4  1 0 0 1 mod 17").unwrap();
+    ///
+    /// let poly_zq = PolyOverZq::from(&modulus);
+    ///
+    /// # let cmp_poly = PolyOverZq::from_str("4  1 0 0 1 mod 17").unwrap();
+    /// # assert_eq!(cmp_poly, poly_zq);
```

Comment:
```suggestion
    /// let cmp_poly = PolyOverZq::from_str("4  1 0 0 1 mod 17").unwrap();
    /// assert_eq!(cmp_poly, poly_zq);
```
I think that this test helps to understand the function better.